voluptuous/schema_builder.py

```diff
@@ -148,11 +150,24 @@ def Extra(_) -> None:
 primitive_types = (bool, bytes, int, str, float, complex)
 
 Schemable = typing.Union[
-    'Schema', 'Object',
+    "Schema",
+    "Object",
     collections.abc.Mapping,
-    list, tuple, frozenset, set,
-    bool, bytes, int, str, float, complex,
-    type, object, dict, None, typing.Callable
+    list,
+    tuple,
+    frozenset,
+    set,
+    bool,
+    bytes,
+    int,
+    str,
+    float,
+    complex,
+    type,
+    object,
+    dict,
+    None,
+    typing.Callable,
```

Comment:
this had intentional compact formatting (similar types on the same line), please wrap it in `# fmt: off` and `# fmt: on`

Reply:
Agreed.

Reply:
sure thing